presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3376,12 +3376,12 @@ private void testRowsWithNulls(Session session, HiveStorageFormat format)
     public void testComplex()
     {
         assertUpdate("CREATE TABLE tmp_complex1 AS SELECT " +
-                        "ARRAY [MAP(ARRAY['a', 'b'], ARRAY[2.0E0, 4.0E0]), MAP(ARRAY['c', 'd'], ARRAY[12.0E0, 14.0E0])] AS a",
+                        "ARRAY [MAP(ARRAY['a', 'b'], ARRAY[2, 4.0]), MAP(ARRAY['c', 'd'], ARRAY[12, 14.0])] AS a",
```

Comment:
You changed semantics here.
It used to be array of doubles, now it's array of `decimal(2,1)`

Same in some other places.


Can we drop this commit in order to expedite the merge of the important part of this PR?

Reply:
I dropped this commit. Thanks